modules/aggregation-layers/src/hexagon-layer/hexagon-aggregator.js

```diff
@@ -36,17 +36,26 @@ export function pointToHexbin({data, radius, getPosition}, viewport) {
 
   // add world space coordinates to points
   const screenPoints = [];
+  let position;
   const {iterable, objectInfo} = createIterable(data);
   for (const object of iterable) {
     objectInfo.index++;
-    screenPoints.push(
-      Object.assign(
-        {
-          screenCoord: viewport.projectFlat(getPosition(object, objectInfo))
-        },
-        object
-      )
-    );
+    position = getPosition(object, objectInfo);
+    const arrayIsFinite = Number.isFinite(position[0]) && Number.isFinite(position[1]);
+    if (arrayIsFinite) {
+      screenPoints.push(
+        Object.assign(
+          {
+            screenCoord: viewport.projectFlat(position)
+          },
+          object
+        )
+      );
+    } else {
+      log.warn(
+        'At least one row was ignored for hexbinning because it had a non-finite position value'
```

Comment:
Can we make the message more concise? E.g. `HexagonLayer: invalid position`

Reply:
Addressed in latest change